src/components/ui/InputText/input-text.scss

```diff
@@ -1,27 +1,71 @@
 @import "src/styles/scaffold";
 
 [data-fs-input-text] {
-  --fs-input-text-padding-horizontal: var(--fs-spacing-2);
+  // --------------------------------------------------------
+  // Design Tokens for Input Text
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-input-text-padding               : var(--fs-spacing-2) var(--fs-spacing-2) 0;
+  --fs-input-text-padding-label         : 0 var(--fs-spacing-2);
+
+  --fs-input-text-box-shadow            : none;
+  --fs-input-text-box-shadow-hover      : 0 0 0 var(--fs-border-width-default) var(--fs-border-color-default-active);
+
+  --fs-input-text-border-radius         : var(--fs-border-radius-default);
+  --fs-input-text-border-width          : var(--fs-border-width-default);
+  --fs-input-text-border-color          : var(--fs-border-color-default);
+  --fs-input-text-border-color-hover    : var(--fs-border-color-default-active);
+
+  --fs-input-text-text-color            : var(--fs-color-text);
```

Comment:
Does it suppose to repeat `text` twice here?

Reply:
unfortunately, yes 🥺 the component name is `input-text`